From: NIIBE Yutaka <gniibe@fsij.org>
Date: Fri, 12 Jun 2020 08:59:01 +0900
Subject: build: Fix command-line to make 'src 'dir.

* configure.ac: Mistake, then, fix it.

Fixes-commit: ce97528fa3dcc58d86e18d4d8820f210a624f63d
Signed-off-by: NIIBE Yutaka <gniibe@fsij.org>
(cherry picked from commit 159e394920f6be822d771fe003498de4d93b07c3)
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index c51c7d3..e0a466c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -600,7 +600,7 @@ if test x$cross_compiling = xyes; then
     AC_CHECK_PROG(CROSS_HOST_OBJDUMP, [$host_alias-objdump], [yes])
     if test "$CROSS_HOST_OBJDUMP" = "yes" ; then
       lock_obj_h_generated=yes
-      if test ! -d src; mkdir src; fi
+      if test ! -d src; then mkdir src; fi
       LOCK_ABI_VERSION=1 host=$host host_alias=$host_alias \
           CC=$CC OBJDUMP=$host_alias-objdump \
           ac_ext=$ac_ext ac_objext=$ac_objext \
